### DISCUSSION TITLE
Project Submission Button

### DISCUSSION THREAD

**Discussion by Riya-hiwanj** [github/india#857] [Students]

**Comment by kranthi2996** [comment]
https://www.linkedin.com/in/kranthikumar2996/ -Take my profile as refernce and check whats the mistake

**Reply by prydsn121** [reply]
I have same issue . I  have already added Microsoft Azure as a skill on my Linked In profile and also have mailed several times to support team but still I am not getting the project submit button.
My linked in profile link: https://www.linkedin.com/in/priyadarshan-290073213/
please help me.

**Reply by Diksha4111** [reply]
I'm also facing the same problem. Even after adding Microsoft Azure skill on LinkedIn the project submission button is not visible. I also mailed the support team many times with the screenshots of my LinkedIn profile skill page the submission button wasn't visible.  PLEASE HELP ME 

my LinkedIn profile link : [https://www.linkedin.com/in/diksha-yadav-861bb121a/](url)
